test/e2e/util.go

```diff
@@ -72,7 +72,7 @@ func patchAndCheckService(t *testing.T, client *framework.ClientSet, isOperatorM
 
 func patchAndCheckRoute(t *testing.T, client *framework.ClientSet, isOperatorManaged bool) error {
 	t.Logf("patching TargetPort on the console Route")
```

Comment:
Does this log statement need to be updated as well?

Reply:
Yes, you are right. Im blind 🙈